component/core/src/test/java/org/exoplatform/social/core/jpa/storage/IdentityStorageTest.java

```diff
@@ -441,40 +441,40 @@ public void testGetIdentitiesSorted() throws Exception {
 
     String sortField = SortBy.FULLNAME.getFieldName();
     String fieldName = Profile.FULL_NAME;
-    List<Identity> result = identityStorage.getIdentities(providerId, firstCharacterFieldName, firstCharacter, sortField, sortDirection, offset, limit);
+    List<Identity> result = identityStorage.getIdentities(providerId, firstCharacterFieldName, firstCharacter, sortField, sortDirection, false, offset, limit);
```

Comment:
You need to add dedicated UT for the case excludeExternal is true.

Reply:
Test case added